modules/openapi-generator/src/main/resources/go-server/routers.mustache

```diff
@@ -7,7 +7,9 @@ import (
 	"net/http"
 	"os"
 	"strconv"
+{{#featureCORS}}
 
+	"github.com/gorilla/handlers"{{/featureCORS}}
```

Comment:
What about removing line 11 and use the following instead?
```
{{#featureCORS}}
	"github.com/gorilla/handlers"
{{/featureCORS}}
```
(which will avoid blank lines)